entrypoint.sh

```diff
@@ -18,6 +18,11 @@ SUBSONIC_DEFAULT_MUSIC_FOLDER=${SUBSONIC_DEFAULT_MUSIC_FOLDER:-/var/music}
 SUBSONIC_DEFAULT_PODCAST_FOLDER=${SUBSONIC_DEFAULT_PODCAST_FOLDER:-/var/music/Podcast}
 SUBSONIC_DEFAULT_PLAYLIST_FOLDER=${SUBSONIC_DEFAULT_PLAYLIST_FOLDER:-/var/playlists}
 
+# Create subsonic user, taking uid, gid and homedir from (Docker) environment
+groupadd --system -o --gid "$SUBSONIC_UID" subsonic && \
```

Comment:
Seems fine to include this in the entrypoint instead of the dockerfile. However, what will this do on subsequent starts after the initial start? The user and group will already exist so I would expect this to fail which should cause the entrypoint script to exit with an error signal since I'm using `set -e` in the script.

I think if you just protect creation with a check for user and group existence this would work. I would check them separately rather than chaining user and group creation in that case as well. They were chained in the dockerfile to only create one layer since it was guaranteed they would exist on image creation. Here, there are no filesystem layers created so there's no benefit to chaining.

Reply:
Hi @stuckj, thank you for the quick response.

Actually, no, when the container is re-created, the user won't exist, because it's not created in the Dockerfile anymore and `/etc` is part of the ephemeral container filesystem and not mounted on a persistent volume.
So when the container is created anew, the script will just create the user/group anew as well.

Therefore, adding a check doesn't IMHO make a lot of sense, since that check would always be false.

Reply:
Hang on, sorry, I was distracted and didn't think it through properly - you're right, upon container start, this will endlessly create new users ... 

I'll fix it - sorry again.

Reply:
Ok, I've added checks avoiding the issue.
Sorry for this again, I've hacked it together too quickly, while also doing other stuff on the side.

Anyway, thank you for considering to merge this.